org.opent2t.onboarding.insteon/js/README.md

```diff
@@ -0,0 +1,5 @@
+# Insteon Hub Onboarding
+Sample [Insteon Hub](http://www.insteon.com/) onboarding module for Open Translators to Things. May be used to discover devices connected to a Insteon Hub per the 
+[org.opent2t.onboarding.insteon.xml](https://github.com/opent2t/onboarding/blob/master/org.opent2t.onboarding.insteon/org.opent2t.onboarding.insteon.xml) schema.
+
+See schema definition for inputs to the onboarding method.
```

Comment:
Added.

---
In reply to: [82907511](https://github.com/openT2T/onboarding/pull/21#discussion_r82907511) [](ancestors = 82907511)